src/IIIFPresentation/API/Features/Manifest/CanvasPaintingMerger.cs

```diff
@@ -169,31 +182,35 @@ private void ValidateItemCanvasPainting(InterimCanvasPainting itemsCanvasPaintin
                     $"Canvas painting with id {itemsCanvasPainting.Id} cannot contain an annotation body");
             }
         }
-
-        if (paintedResourceCanvasPainting.CanvasLabel == null && itemsCanvasPainting.CanvasLabel != null)
+        
+        if (paintedResourceCanvasPainting.CanvasLabel == null && 
```

Comment:
Comment explaining? Or can it be extracted into a method? `if (SomethingExplanatory()) { ... }`